torch/_inductor/fx_passes/overlap_manual_scheduling.py

```diff
@@ -110,10 +114,13 @@ def _bucket_group(self, coll_nodes: list[fx.Node]) -> None:
             "bucketed_all_gather" if is_all_gather(first) else "bucketed_reduce_scatter"
         )
         for n in new_nodes:
-            n.meta["nn_module_stack"] = coll_nodes[0].meta.get("nn_module_stack", "")
-            n.meta["fwd_nn_module_stack"] = coll_nodes[0].meta.get(
-                "fwd_nn_module_stack", ""
-            )
+            for key, default in [
+                ("nn_module_stack", ""),
+                ("fwd_nn_module_stack", ""),
+                ("custom", {}),
+                ("stack_trace", ""),
+            ]:
+                n.meta[key] = coll_nodes[0].meta.get(key, default)
```

Comment:
Can we make bucketing.py peserve this instead of overlap_manual_scheduling? this is generically useful.